pybikes/telofun.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017, aronsky <aronsky@gmail.com>
+
+from xml.dom import minidom
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class TeloFun(BikeShareSystem):
+    sync = True
+    unifeed = True
+
+    meta = {
+        'system': 'telofun',
+        'company': ['FSM Ground Services Ltd.']
+    }
+
+    def __init__(self, tag, meta, feed_url):
+        super(TeloFun, self).__init__(tag, meta)
+
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        scraper = scraper or utils.PyBikesScraper()
+
+        stations = []
+
+        data = scraper.request(self.feed_url)
+        dom = minidom.parseString(data.encode('utf-8', errors='ignore'))
+        stations = self.get_stations(dom)
+        self.stations = list(stations)
+
+    def get_stations(self, dom):
+        for placemark in dom.getElementsByTagName('Placemark'):
+            name = placemark.getElementsByTagName('name')[0].childNodes[0] \
+                   .nodeValue
+            info = placemark.getElementsByTagName('description')[0] \
+                   .childNodes[0].nodeValue
```

Comment:
Whilst using minidom is absolutely ok, sometimes it can result on a involved solution. See same implementation using lxml:

```
from lxml import etree
...
ns = {'kml': 'http://www.opengis.net/kml/2.2'}
dom = etree.fromstring(data.encode('utf-8'))
placemarks = dom.xpath('//Placemark', namespaces=ns)
for placemark in placemarks:
    name = placemark.findtext('kml:name', namespaces=ns)
    ...
```

Since minidom is part of the standard library the decision of using lxml or minidom is up to you, just though it was a good FYI